flang/runtime/io-api.cpp

```diff
@@ -1401,6 +1401,12 @@ void IONAME(GetIoMsg)(Cookie cookie, char *msg, std::size_t length) {
   }
 }
 
+AsynchronousId IONAME(GetAsynchronousId)(Cookie cookie) {
+  // Flang runtime implements asynchronous IO synchronously.
```

Comment:
The I/O runtime creates and tracks asynchronous operation IDs as part of the state of any external I/O statement so that `WAIT` statement semantics work correctly and detect errors.  Please return the value of `asynchronousID()` from the supplied cookie.

Reply:
Thanks, updated.